src/app/ubs/ubs/components/ubs-order-details/ubs-order-location-popup/ubs-order-location-popup.component.ts

```diff
@@ -44,6 +44,7 @@ export class UbsOrderLocationPopupComponent implements OnInit, OnDestroy {
   }
 
   filterOptions(): void {
+    this.myControl.setValue({ locationId: 1, locationName: 'Kyiv, Kyiv Oblast' });
```

Comment:
1) Why here do we directly set Kyiv by the name of location?
2) Also here we can get the locationId from back-end, because if this value changes, we will get an error on the front. At least, creating a separate constant for these item is better and use KyivNamesEnum for 'Kyiv, Kyiv Oblast'.
3) Due to the point 1 we will get a wrong value in 'UA' mode 
![image](https://github.com/ita-social-projects/GreenCityClient/assets/101433204/0a761e75-c716-4f49-a593-73b98a01a357)